exercises/00-init/src/react-app-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```

Comment:
Why is this necessary? I'm just curious because I don't use CRA very often. Actually, I haven't used it since this training. :smiley:

Reply:
It suppose to reference the react-script types to help with some imports.

Either way, it's automatically generated when I run `npm start`. Even if you delete the file it will be created again (possibly meaning, that it's important?).

I'm unsure how to proceed with this one.

Reply:
OK, no problem to keep it. Or we could add it to `.gitignore` if it's generated because future versions of CRA might not need it.